ChangeLog.md

```diff
@@ -1,6 +1,17 @@
 
 # Change Log
 
+## Version 10.3.61 (private drop)
+
+### Performance optimizations
+
+1. Any `--include-pattern` that contains only `*` wildcards will be performance optimized when querying blob storage.  The section before the 
+first `*` will be used as a server-side prefix, to filter the search results more efficiently. E.g. "--include-path abc*" will be implemented 
+as a prefix search for "abc". In a more complex example, "--include-path abc\*123", will be implemented as a prefix search for "abc", followed
+by client-side filtering to find exact matches to abc\*123.
+2. When processing over a million files, AzCopy will report on its progress once ever 2 minutes instead of once every 2 seconds.  This reduces the CPU 
```

Comment:
This is inaccurate information. Because the \ is present, we'd ignore attempting to prefilter, because we think it's too complex. This isn't a necessary change right now (we can do it when we get to the rest of this PR, or when we update the changelogs otherwise)